package.json

```diff
@@ -17,13 +17,19 @@
   },
   "devDependencies": {
     "@continuous-auth/semantic-release-npm": "^3.0.0",
+    "@types/github-url-to-object": "^4.0.1",
+    "@types/is-url": "^1.2.30",
+    "@types/ms": "^0.7.31",
+    "electron": "^22.0.0",
```

Comment:
question: how come we needed to add Electron as a dev dep? Does this need to be updated every time there's a new major release of Electron?